db/schema.rb

```diff
@@ -11,15 +11,30 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160127035545) do
+ActiveRecord::Schema.define(version: 20160127193752) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "games", force: :cascade do |t|
```

Comment:
Never, because it's automatically generated. Any changes you make will just be overridden next time anyone runs any migration.

You'll have to do an interactive git rebase to solve it probably.